SystemInformer/memrslt.c

```diff
@@ -315,10 +315,7 @@ INT_PTR CALLBACK PhpMemoryResultsDlgProc(
             {
                 RECT rect;
 
-                rect.left = 0;
-                rect.top = 0;
-                rect.right = 250;
-                rect.bottom = 180;
+                SetRect(&rect, 0, 0, 250, 180);
```

Comment:
Convention elsewhere in the project avoids `SetRect`. I suggest reverting this change. And following convention elsewhere (don't use `SetRect`).